library/std/src/env.rs

```diff
@@ -350,18 +392,23 @@ impl Error for VarError {
 /// # Examples
 ///
 /// ```
+/// # #![allow(unused_unsafe)]
+///
```

Comment:
```suggestion
```
or else the empty line would show in the docs

Reply:
Done.